### DISCUSSION TITLE
Markdown Carriage return

### DISCUSSION THREAD

**Discussion by colorao** [directus/directus#14999] [Feature Requests]
I've realized that the Markdown editor has neither "carrriage return" nor "non breakable space" icon/shortcut.
I know that both could be added by typing the proper html entity or with tricks (double space+nl or backslash+nl) but I find it not intuitive for any final user (clients, editors or translators) to memorize and type such tricky combinations. 

I suggest to add 2 icons (+ shortcuts) : CR and NBSP which are common and fundamental in the edition world.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
